front/src/applications/operationalStudies/views/Project.jsx

```diff
@@ -194,7 +193,7 @@ export default function Project() {
                 </div>
                 <div className="project-details-financials-amount">
                   <span className="project-details-financials-amount-text">{t('totalBudget')}</span>
-                  {budgetFormat(project.budget)}
```

Comment:
you need to fix the `budgetFormat` function.

can you create a unit test for each of the cases you want work ?